agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/LitDocument.java

```diff
@@ -0,0 +1,19 @@
+package org.alliancegenome.indexer.indexers.curation;
+
+import org.alliancegenome.curation_api.model.document.es.ESDocument;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.gson.JsonObject;
+
+import lombok.Data;
+
+@Data
+public class LitDocument extends ESDocument {
```

Comment:
If you can spell out this document as in LiteratureSummaryDocument that should be its name.

Reply:
@oblodgett yes, rename to LiteratureSummaryDocument, also rename LitIndex to LiteratureIndex